src/audio/eq_iir/eq_iir.c

```diff
@@ -504,15 +512,14 @@ static int eq_iir_init_coef(struct sof_eq_iir_config *config,
 			/* Initialize EQ channel to bypass and continue with
 			 * next channel response.
 			 */
-			comp_cl_info(&comp_eq_iir, "eq_iir_init_coef(), ch %d is set to bypass",
-				     i);
+			comp_info(mod->dev, "eq_iir_init_coef(), ch %d is set to bypass", i);
 			iir_reset_df1(&iir[i]);
 			continue;
 		}
 
 		if (resp >= config->number_of_responses) {
-			comp_cl_info(&comp_eq_iir, "eq_iir_init_coef(), requested response %d exceeds defined",
-				     resp);
+			comp_info(mod->dev, "eq_iir_init_coef(), requested response %d exceeds defined",
```

Comment:
shouldn't this be at least a warning, or even an error, since it returns an error code?

Reply:
Yep, good catch.